website/docs/r/nsxt_network_imported.html.markdown

```diff
@@ -107,13 +133,23 @@ and inherited from provider configuration)
 -> One of `nsxt_logical_switch_name` or `dvpg_name` must be provided.
 
 * `description` - (Optional) An optional description of the network
-* `gateway` - (Required) The gateway for this network (e.g. 192.168.1.1)
+* `gateway` - (Required) The gateway for this network (e.g. 192.168.1.1, 2002:0:0:1234:abcd:ffff:c0a7:121)
```

Comment:
I did not go the explicit handling way.

> I changed one of the tests start IP addresses with an additional 0 in the second group and it returns a cryptic error

Could you share more exactly what did you set and where so that I can try?

Reply:
On line 90 I changed start address from
```
"start_address": "2002:0:0:1234:abcd:ffff:c0a6:122",
```
to

```
"start_address": "2002:00:0:1234:abcd:ffff:c0a6:122"
```

Reply:
> On line 90 I changed start address from

Line 90 is a "check", not a config snippet - was it a config snippet that you changed?

I tried to change config snippet in line 386 - it did work for me. 
```
=== RUN   TestAccVcdIpv6Support
    ipv6_test.go:44: Step 1/1 error: Check failed: 1 error occurred:
                * Check 42/54 error: "vcd_network_isolated_v2.ipv6-dualstack" no TypeSet element "secondary_static_ip_pool.*", with nested attrs map[string]string{"end_address":"2002:0:0:1234:abcd:ffff:c0a6:124", "start_address":"2002:0:0:1234:abcd:ffff:c0a6:122"} in state: map[string]string{"%":"19", "description":"", "dns1":"", "dns2":"", "dns_suffix":"", "dual_stack_enabled":"true", "gateway":"192.168.1.1", "id":"urn:vcloud:network:4de9cb06-aeb7-41ba-80ad-7fbd6d51beaa", "is_shared":"false", "metadata.%":"0", "metadata_entry.#":"0", "name":"TestAccVcdIpv6Support-isolated-dualstack", "org":"datacloud", "owner_id":"urn:vcloud:vdc:da586866-2981-4084-9ede-5f26f2ca8b40", "prefix_length":"24", "secondary_gateway":"2002:0:0:1234:abcd:ffff:c0a6:121", "secondary_prefix_length":"124", "secondary_static_ip_pool.#":"1", "secondary_static_ip_pool.0.%":"2", "secondary_static_ip_pool.0.end_address":"2002:0:0:1234:abcd:ffff:c0a6:124", "secondary_static_ip_pool.0.start_address":"2002:00:0:1234:abcd:ffff:c0a6:122", "static_ip_pool.#":"1", "static_ip_pool.0.%":"2", "static_ip_pool.0.end_address":"192.168.1.20", "static_ip_pool.0.start_address":"192.168.1.10", "vdc":"TestAccVcdIpv6Support"}
```

In the above example, it was looking for `"start_address":"2002:0:0:1234:abcd:ffff:c0a6:122"` (because the test check is made so), but got `.start_address":"2002:00:0:1234:abcd:ffff:c0a6:122"` so it does look fine.

I have tried to perform Go validation for that IP (https://go.dev/play/p/J5Yo90hmnpe). It is a valid IP.

Reply:
Sorry, it was a quirk on my end regarding the test error. My point was that the API returns the IP in `1111:0:0:0:0:0:0:1111` format, and if someone passes `1111::1111` it will result in an inconsistent plan, so maybe a note or additional information in the argument reference would help others.

Reply:
Ok. So lets check on the note. 

How does something like that look for affected resources 
<img width="687" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/23108a9f-fb07-4d6a-ac8b-dda1bd07a177">

Based on a real test `~ secondary_gateway       = "2002:0:0:1234:abcd:ffff:c0a6:121" -> "2002::1234:abcd:ffff:c0a6:121"`

Reply:
That works for me, thanks!

Reply:
Just confirming that I added these notes to related resources.